ui/app/styles/clinical/_program.scss

```diff
@@ -254,4 +254,63 @@
     height: 120px;
     margin-bottom: 10px;
   }
-}
\ No newline at end of file
+}
+
+/*scroll investigation chart table*/
```

Comment:
Hi @shilpa-iplit ,I think you should remove the comment.Apart from this,everything looks fine